x-pack/plugins/ml/public/data_frame/common/index.ts

```diff
@@ -173,3 +225,4 @@ export function getDataFrameRequest(
 }
 
 export * from './index_pattern_context';
+export * from './pivot_group_by';
```

Comment:
as this index file has grown, should all the code be moved to a separate file and the index just have the `export` statements?
This is the pattern we normally use and has been used later on in this PR

Reply:
👍 did the refactor in e635e14.